source/hardware/controllers/native_instruments_traktor_kontrol_s3.rst

```diff
@@ -0,0 +1,165 @@
+Native Instruments Traktor Kontrol S3
+=====================================
+
+.. sectionauthor::
+  Owen Williams <owilliams at mixxx.org>
+
+.. versionadded:: 2.3.0
+
+The Kontrol S3 is an introductory 4 deck controller with good build
+quality and integrated sound card. This is the first controller released
+with the "S3" name.
+
+The Kontrol S3 can run from USB bus power. Using the separate power
+supply increases the brightness of the LEDs, which is helpful for using
+it in daylight, and increases the volume of the headphone output.
+
+  - `Manufacturer's product
+    page <https://www.native-instruments.com/en/products/traktor/dj-controllers/traktor-kontrol-s3/>`__
+
+Compatibility
+-------------
+
+Controller
+~~~~~~~~~~
+
+The Kontrol S3 is a USB class compliant audio and :term:`HID` device,
+so it is compatible with Mixxx without any proprietary drivers on
+GNU/Linux and Mac OS X. On Windows, it is recommended to install the
+`driver from Native
+Instruments <https://www.native-instruments.com/en/support/downloads/drivers-other-files/>`__
+and select the ASIO sound API in the Sound Hardware section of Mixxx's
+Preferences.
+
+With the S3 plugged in, a MIDI device is listed as an available
+controller in Mixxx's Preferences. The controller uses
```

Comment:
Was this copied and pasted from another wiki page? Is this true for the S3? I didn't think it had MIDI I/O.

Reply:
Good catch.  (It does have a midi mode if you hold some buttons during startup, but that's not the default)